angularjs/angular-animate.d.ts

```diff
@@ -250,11 +256,16 @@ declare namespace angular.animate {
         /**
          * The numeric index representing the stagger item (e.g. a value of 5 is equal to the sixth item
          * in the stagger; therefore when a stagger option value of 0.1 is used then there will be a stagger delay of 600ms)
-         * applyClassesEarly - Whether or not the classes being added or removed will be used when detecting the animation.
```

Comment:
looks like `applyClassesEarly` is a valid field in this interface -- would you mind adding it back as a proper field (not a comment)?